tests/console/yast2_vnc.pm

```diff
@@ -46,13 +46,13 @@ sub configure_remote_admin {
 
 sub check_service_listening {
     my $cmd_check_port = $is_older_product ? 'netstat' : 'ss -tln | grep -E LISTEN.*:5901';
-    if (script_run($cmd_check_port)) {
-        record_soft_failure 'boo#1088646 - service vncmanager is not started automatically';
-        systemctl('status vncmanager', expect_false => 1);
-        systemctl('restart vncmanager');
-        systemctl('status vncmanager');
-        assert_script_run $cmd_check_port;
+    my $is_open;
+    for (0 .. 5) {    # try several times
+        $is_open = 1 if !script_run($cmd_check_port);
+        last if ($is_open);
```

Comment:
Do you think adding some break (e.g. 1 second sleep) between the attempts should help?

Reply:
I thought it's of like that, as the time for typing five times probably takes several seconds. It's hard to tell a we cannot reproduce the issue, I'm not against adding a sleep if you think it'd help.

Reply:
Yep, I also thought typing already creates some delay, but sleep won't hurt I guess in case we switch this test to serial console where we don't have typing delay.